llvm/lib/CodeGen/TargetPassConfig.cpp

```diff
@@ -272,6 +272,12 @@ static cl::opt<bool>
                     cl::desc("Split static data sections into hot and cold "
                              "sections using profile information"));
 
+/// Enable matching and inference when using propeller.
+static cl::opt<bool>
+    PropellerMatchInfer("propeller-match-infer",
+                        cl::desc("Use match&infer to evaluate stale profile"),
```

Comment:
The description ("...evaluate...") doesn't match the comment ("enable"); adjust maybe?